packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -76,21 +101,45 @@ class FlutterGoldenFileComparator implements GoldenFileComparator {
 
   @override
   Future<bool> compare(Uint8List imageBytes, Uri golden) async {
+    if(_testingWithSkiaGold()) {
+      // Library prefix notation for Skia Gold.
```

Comment:
Not sure what you mean by this and the comment below. Maybe remove these or be more specific?